react/components/atoms/avatar/avatar.js

```diff
@@ -60,9 +69,13 @@ export class Avatar extends mix(PureComponent).with(IdentifiableMixin) {
                 hitSlop={this.props.hitSlop}
             >
                 <Image
-                    source={this.props.image}
+                    source={this.state.imageSrc}
                     style={styles.image}
                     resizeMode={this.props.resizeMode}
+                    onError={() => {

```

Comment:
Extract to `onError` function

Reply:
I'm afraid I cannot access` this.state` in `defaultProps()` function

Unless I missed something, you asked to define this directly in `onError` prop